src/proposals/20250529/SparkEthereum_20250529.t.sol

```diff
@@ -0,0 +1,784 @@
+            }),
+            currentCap: 500_000_000e6,
+            newCap:     1_000_000_000e6
+        });
+    }
+
+    function test_ETHEREUM_WBTCChanges() public onChain(ChainIdUtils.Ethereum()) {
+        SparkLendContext memory ctx = _getSparkLendContext();
+
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', ctx.pool);
+        ReserveConfig memory config = _findReserveConfigBySymbol(allConfigsBefore, 'WBTC');
+
+        assertEq(config.liquidationThreshold, 45_00);
+
+        executeAllPayloadsAndBridges();
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', ctx.pool);
+        
+        config.liquidationThreshold = 40_00;
+
+        _validateReserveConfig(config, allConfigsAfter);
+    }
+
+    function test_ETHEREUM_sllCoreRateLimitIncrease() public onChain(ChainIdUtils.Ethereum()) {
+        bytes32 usdsMintKey       = MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_USDS_MINT();
+        bytes32 swapUSDSToUSDCKey = MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_USDS_TO_USDC();
+        bytes32 ethenaMintKey     = MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_USDE_MINT();
+        bytes32 ethenaBurnKey     = MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_USDE_BURN();
+        bytes32 susdeCooldownKey  = MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_SUSDE_COOLDOWN();
+        
+        bytes32 susdeDepositKey   = RateLimitHelpers.makeAssetKey(MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_4626_DEPOSIT(), Ethereum.SUSDE);
+        bytes32 susdeWithdrawKey  = RateLimitHelpers.makeAssetKey(MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_4626_WITHDRAW(), Ethereum.SUSDE);
+        
+        _assertRateLimit(usdsMintKey,       200_000_000e18, 200_000_000e18 / uint256(1 days));
+        _assertRateLimit(swapUSDSToUSDCKey, 200_000_000e6,  200_000_000e6 / uint256(1 days));
+        _assertRateLimit(ethenaMintKey,     50_000_000e6,   50_000_000e6 / uint256(1 days));
+        _assertRateLimit(ethenaBurnKey,     100_000_000e18, 100_000_000e18 / uint256(1 days));
+        _assertRateLimit(susdeDepositKey,   100_000_000e18, 100_000_000e18 / uint256(1 days));
+        _assertRateLimit(susdeCooldownKey,  500_000_000e18, 250_000_000e18 / uint256(1 days));
+
+        executeAllPayloadsAndBridges();
+
+        _assertRateLimit(usdsMintKey,       500_000_000e18, 500_000_000e18 / uint256(1 days));
+        _assertRateLimit(swapUSDSToUSDCKey, 500_000_000e6,  300_000_000e6 / uint256(1 days));
+        _assertRateLimit(ethenaMintKey,     250_000_000e6,  100_000_000e6 / uint256(1 days));
+        _assertRateLimit(ethenaBurnKey,     500_000_000e18, 200_000_000e18 / uint256(1 days));
+        _assertRateLimit(susdeDepositKey,   250_000_000e18, 100_000_000e18 / uint256(1 days));
+        
+        _assertUnlimitedRateLimit(susdeCooldownKey);
+        _assertUnlimitedRateLimit(susdeWithdrawKey);
+    }
+
+    function _testRateTargetBaseToKinkIRMUpdate(
```

Comment:
Maybe makes sense to move it to https://github.com/sparkdotfi/spark-spells/blob/sc-993-spell-2025-05-29/src/test-harness/SparkEthereumTests.sol ?

Reply:
discussed this with Lucas, it would not be used anywhere after this therefore kept in the tests